cve_bin_tool/version_signature.py

```diff
@@ -9,14 +9,22 @@
 from cve_bin_tool.cvedb import DISK_LOCATION_DEFAULT
 
 
+class InvalidVersionSignatureTable(ValueError):
+    """Raised when an invalid table name is given to version_signature"""
+
+
 class VersionSignatureDb:
     """Methods for version signature data stored in sqlite"""
 
     def __init__(self, table_name, mapping_function, duration) -> None:
         """Set location on disk data cache will reside.
         Also sets the table name and refresh duration
         """
+        if not table_name.isalnum():
+            # Basic validation here so we can safely ignore Bandit SQL warnings
+            raise InvalidVersionSignatureTable
```

Comment:
Might be helpful to include the name so it's obvious to the user what might be invalid, since we're at it.

```suggestion
            raise InvalidVersionSignatureTable(repr(table_name))
```